### ISSUE TITLE
mch acl functions in os_unix.c

### ISSUE THREAD

**Issue by Adam-K-P** [neovim/neovim#3819]
Hello, I was looking through os_unix.c and as I understand it, the mch_* functions are being ported to libuv.  

I was looking, specifically, at:
mch_free_acl
mch_get_acl
mch_set_acl

I'm having some trouble figuring out exactly what these functions are doing.  It looks like all the system calls from the original vim source have been removed, so are these functions really doing anything?

These mch_ functions are being called in undo.c and fileio.c.

Basically, I'm wondering what should be done with these functions?  Should they just be left alone?  Or should they be removed along with all the ifdef HAVE_ACL code?

I found this issue being discussed here: https://github.com/neovim/neovim/issues/1200

However, there are no ACL functions in libuv from what I can tell that these can be ported to.

**Comment by justinmk** [comment]
Investigating and answering those questions is probably 80% of the work needed to address https://github.com/neovim/neovim/issues/1200.
